charts/dynatrace-oneagent-operator/v0.9.2/templates/Kubernetes/rolebinding-oneagent-unprivileged.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Dynatrace LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- $platformIsSet := printf "%s" (required "Platform needs to be set to kubernetes or openshift" (include "dynatrace-oneagent-operator.platformSet" .))}}
+{{- if eq .Values.platform "kubernetes" }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: dynatrace-oneagent-unprivileged
+  namespace: dynatrace
```

Comment:
Hardcoded namespaces like this will need to be templated. This causes the installation to fail because this namespace doesn't get created by Rancher.

Reply:
Updated the yaml to use the given helm namespace instead